### PR TITLE
fixes logo distortion on MainHeader.js and repositions UploadCase button next to the account icon

### PR DESCRIPTION
**Pull request by jglez**
# Navigation bar UI Fixes

## What was changed
- The human rights first logo is no longer distorted
- The "Upload A Case" button was repositioned next to the account icon as discussed during product reviews
old-nav
nav-fix

## Type of change
- Bug fix/UI fix

## For the reviewers
The height of `MainHeader.js` was slightly decreased as a side effect. Personally, I think it looks better as the nav looked bulky before, but I will leave it to the design leads to approve or decide to keep the previous height. @nicholasbus @CatRadin @emmac124 

Next, I want to increase the size of the account icon logo as it looks a bit too small currently.

### CODE REVIEW

**Review by fullstackcaveman — APPROVED** [review]
Looks good!

File: src/components/pages/Home/MainHeader.js [hunk 1]
```diff
@@ -35,14 +35,16 @@ export default function MainHeader(props) {
...
       <img src={HRFLogo} alt="HRF Logo" width="350px" />
-      <UploadCase getPendingCases={getPendingCases} />
-      <Dropdown className="account-drop-down" overlay={menu}>
-        <ul className="accounts" onClick={e => e.preventDefault()}>
-          <div className="user-button">
-            <Icon component={() => <img src={Profile} alt="profile icon" />} />
-          </div>
-        </ul>
-      </Dropdown>
+      <div id="nav-flex-container">
```
Comment (nicholasbus) [inline src/components/pages/Home/MainHeader.js:38]: Like the id naming convention!

File: src/components/pages/Home/_MainHeader.less [hunk 2]
```diff
@@ -37,6 +37,10 @@
   justify-content: space-between;
 }
 
+#nav-flex-container {
+  display: flex;
```
Comment (nicholasbus) [inline src/components/pages/Home/_MainHeader.less:41]: Nice short and simple fix 👍

**Review by nicholasbus — APPROVED** [review]
Everything looks good, I like the short and simple fix 👍

**Pull request merged**